sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -830,13 +851,32 @@ export class ServiceBusMessage implements ReceivedMessage {
       return;
     }
     const receiver = this._context.getReceiver(this.delivery.link.name, this.sessionId);
-    throwIfMessageCannotBeSettled(receiver, DispositionType.complete, this.delivery.remote_settled);
+    throwIfMessageCannotBeSettled(
+      receiver,
+      DispositionType.complete,
+      this.delivery.remote_settled,
+      this.sessionId
+    );
 
     return receiver!.settleMessage(this, DispositionType.complete);
   }
   /**
    * The lock held on the message by the receiver is let go, making the message available again in
    * Service Bus for another receive operation.
+   *
+   * - Throws `SessionLockLostError` (for messages from a Queue/Subscription with sessions enabled)
```

Comment:
@throws in documentation?

Reply:
Keeping this format for GA, as all other user facing APIs in the library are using this format. It renders well in out docs as a bulleted list.

Logged https://github.com/Azure/azure-sdk-for-js/issues/2853 to track the discussion around moving to using @throws and the follow up work